stories/infinite-hits.stories.ts

```diff
@@ -17,33 +18,11 @@ storiesOf('Results|InfiniteHits', module)
       ]);
     })
   )
-  .add(
```

Comment:
I removed this story because it was not working anymore and we unit test this behavior anyway.